packages/manager/modules/netapp/src/dashboard/index/translations/Messages_fr_FR.json

```diff
@@ -31,5 +31,36 @@
   "netapp_dashboard_service_configuration_create_volume": "Créer un volume",
-  "netapp_dashboard_configuration_restrictions_info": "Doit uniquement contenir des nombres, lettres, underscores, tirets ou points et un maximum de 64 caractères."
+  "netapp_dashboard_configuration_restrictions_info": "Doit uniquement contenir des nombres, lettres, underscores, tirets ou points et un maximum de 64 caractères.",
+  "netapp_dashboard_location_eu-west-par": "Europe (France - Paris)",
+  "netapp_dashboard_location_eu-west-gra": "Europe (France - Gravelines)",
+  "netapp_dashboard_location_eu-west-rbx": "Europe (France - Roubaix)",
+  "netapp_dashboard_location_eu-west-sbg": "Europe (France - Strasbourg)",
+  "netapp_dashboard_location_eu-west-lim": "Europe (Germany - Limburg)",
+  "netapp_dashboard_location_eu-central-waw": "Europe (Poland - Warsaw)",
+  "netapp_dashboard_location_eu-west-eri": "Europe (UK - Erith)",
+  "netapp_dashboard_location_us-east-vin": "North America (US - East - Vinthill)",
+  "netapp_dashboard_location_us-west-hil": "North America (US - West - Hillsboro)",
+  "netapp_dashboard_location_ca-east-bhs": "North America (Canada - East - Beauharnois)",
+  "netapp_dashboard_location_ap-southeast-sgp": "Asia Pacific (Singapore - Singapore)",
+  "netapp_dashboard_location_ap-southeast-syd": "Asia Pacific (Australia - Sydney)",
+  "netapp_dashboard_location_eu-west-rbx-snc": "Europe (France - Roubaix) (snc)",
+  "netapp_dashboard_location_eu-west-sbg-snc": "Europe (France - Strasbourg) (snc)",
+  "netapp_dashboard_location_ca-east-tor": "North America (Canada - East - Toronto)",
+  "netapp_dashboard_location_ap-south-mum": "Asia Pacific (India - Mumbai)",
+  "netapp_dashboard_location_labeu-west-1-preprod": "LABEU (France - Croix) (preprod)",
+  "netapp_dashboard_location_labeu-west-1-dev-2": "LABEU (France - Croix) (dev-2)",
+  "netapp_dashboard_location_labeu-west-1-dev-1": "LABEU (France - Croix) (dev-1)",
+  "netapp_dashboard_location_eu-west-lz-bru": "Europe (Belgium - Brussels) (lz)",
+  "netapp_dashboard_location_eu-west-lz-mad": "Europe (Spain - Madrid) (lz)",
+  "netapp_dashboard_location_eu-west-gra-snc": "Europe (France - Gravelines) (snc)",
+  "netapp_dashboard_location_us-east-lz-dal": "North America (US - East - Dallas)",
+  "netapp_dashboard_location_us-west-lz-lax": "North America (US - West - Los Angeles)",
+  "netapp_dashboard_location_us-east-lz-chi": "North America (US - East - Chicago)",
+  "netapp_dashboard_location_us-east-lz-nyc": "North America (US - East - New York)",
+  "netapp_dashboard_location_us-east-lz-mia": "North America (US - East - Miami)",
+  "netapp_dashboard_location_us-west-lz-pao": "North America (US - West - Palo Alto)",
+  "netapp_dashboard_location_us-west-lz-den": "North America (US - West - Denver)",
+  "netapp_dashboard_location_us-east-lz-atl": "North America (US - East - Atlanta)",
+  "netapp_dashboard_location_eu-west-lz-mrs": "Europe (France - Marseille)"
```

Comment:
Can those be in common with PR https://github.com/ovh/manager/pull/14050/files at least ?

Reply:
Okay I am doing it on this base PR.